hrjobcontract/js/src/job-contract/controllers/contract-list.js

```diff
@@ -196,6 +197,7 @@ define([
                       if (!result.is_error) {
                         ContractService.updateHeaderInfo();
                         removeContractById($scope.contractCurrent, contractId) || removeContractById($scope.contractPast, contractId);
+                        pubSubService.publish("contract-refresh");
```

Comment:
Single quotes. Also, in order to highlight this, it would make this more readable if you put a newline before it
```js
ContractService.updateHeaderInfo();
removeContractById($scope.contractCurrent, contractId) || removeContractById($scope.contractPast, contractId);

pubSubService.publish('contract-refresh');
```